src/library/dao/playlistdao.cpp

```diff
@@ -458,7 +458,7 @@ void PlaylistDAO::removeTrackFromPlaylist(int playlistId, int position) {
 
 void PlaylistDAO::removeTracksFromPlaylist(int playlistId, const QList<int>& positions) {
     // get positions in reversed order
-    QList sortedPositons = positions;
+    auto sortedPositons = positions;
```

Comment:
Unrelated?

Reply:
This was necessary to build on macOS 10.13 with an older compiler. I could change it to `QList<int>` if you prefer. Either works.